DOC_CHANGES.md

```diff
@@ -6,19 +6,24 @@ Example Doc Change:
 Description of the required change.
 -->
 
-### Resources must now use `provides` to declare recipe DSL
+### Resources must now use `resource_name` (or `provides`) to declare recipe DSL
 
 Resources declared in `Chef::Resource` namespace will no longer get recipe DSL
-automatically.  Instead, explicit `provides` is required in order to have DSL:
+automatically.  Instead, `resource_name` is required in order to have DSL:
```

Comment:
Line above says `resource_name` or `provides`, this line only says `resource_name`.